src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,14 @@
 Release History
 ===============
 
+**IoT Central**
+
```

Comment:
Because our current pipeline automatically generate history notes from PR tile and descriptions, so please don't write the history into history note manually, it should be written into the PR description instead.
If you want more details, please refer to the doc: [link](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests)